#!/usr/bin/env python

PACKAGE = 'jsk_pcl_ros_utils'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#        variable  type  level  description  default  min  max
gen.add("use_likelihood", bool_t, 0, "use likelihood field to select plane", False)
gen.add("plane_index", int_t, 0, "select plane with index", 0, 0, 10)
exit(gen.generate(PACKAGE, "jsk_pcl_ros_utils", "PolygonArrayUnwrapper"))
